pyomo/contrib/mindtpy/initialization.py

```diff
@@ -144,7 +149,11 @@ def init_max_binaries(solve_data, config):
     opt = SolverFactory(config.mip_solver)
     if isinstance(opt, PersistentSolver):
         opt.set_instance(m)
-    results = opt.solve(m, options=config.mip_solver_args)
+    mip_args = dict(config.mip_solver_args)
+    if config.mip_solver == 'gams':
+        mip_args['add_options'] = mip_args.get('add_options', [])
+        mip_args['add_options'].append('option optcr=0.01;')
```

Comment:
For convergence to be guaranteed we actually require the MILP to be solved to optimality (`optcr=0.0`)